downstream/assemblies/eda/assembly-eda-event-filter-plugins.adoc

```diff
@@ -0,0 +1,47 @@
+[id="eda-event-filter-plugins"]
+
+= Event filter plugins
+
+Events sometimes have extra data that is unnecessary and might overwhelm the rule engine. 
+Use event filters to remove that extra data so you can focus on what matters to your rules. 
+Event filters might also change the format of the data so that the rule conditions can better match the data.
+
+Events are defined as python code and distributed as collections. 
+The default link:https://github.com/ansible/event-driven-ansible/tree/main/extensions/eda/plugins/event_filter[eda collection] has the following filters:
+
+[cols="30%,30%",options="header"]
+|====
+| Name | Description
+| json_filter | Include and exclude keys from the event object
+| dashes_to_underscores | This filter changes the dashes in all keys in the payload to be underscore
```

Comment:
@EMcWhinn We have 2 more filters defined in here 
https://github.com/ansible/event-driven-ansible/tree/main/extensions/eda/plugins/event_filter

1. ansible.eda.insert_hosts_to_meta is used to add host information into the event so that ansible-rulebook can locate it and use it.
2. ansible.eda.normalize_keys is used if you want to change non alpha numeric keys to underscore.

Reply:
@mkanoor Thanks for flagging, I've added those 2 filters.